src/Visitors/AddArrayPropertyItem.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace RonasIT\Larabuilder\Visitors;
+
+use PhpParser\Node;
+use PhpParser\Node\ArrayItem;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\PropertyItem;
+use PhpParser\Node\Stmt\Property;
+use PhpParser\Node\Stmt\PropertyProperty;
+use RonasIT\Larabuilder\Exceptions\UnexpectedPropertyTypeException;
+
+class AddArrayPropertyItem extends SetPropertyValue
+{
+    protected ArrayItem $arrayItem;
+    protected PropertyItem $propertyItem;
+    protected Identifier $typeIdentifier;
```

Comment:
not sure we need to redefine them in this class as they already defined in the parent